tests/unit/backend/layers/api/test_curation_api.py

```diff
@@ -475,6 +475,7 @@ def test__verify_expected_public_collection_fields(self):
         resp_collection = resp[0]
         self.check_fields(EntityColumns.link_cols, resp_collection["links"][0], "links")
         self.assertEqual(public_collection.datasets[0].dataset_id.id, resp_collection["datasets"][0]["dataset_id"])
+        self.expected_collection_columns.remove("processing_status")
```

Comment:
Regarding the 'expected_columns' assertion pattern more generally: this is fine for now but I'm thinking we should really be taking care of this in the connexion yml, as required fields, right?

Reply:
I've noticed that we basically defaulted to not requiring any fields.

Reply:
That would be nice if we could read the yaml and know what fields should be returned, but that would be a significant refactor. If we have time we should go that route.